lisa/tools/docker.py

```diff
@@ -137,6 +142,15 @@ def _install(self) -> bool:
             self.node.os.install_packages(["moby-engine", "moby-cli"])
         elif isinstance(self.node.os, SLES):
             self.node.os.install_packages(["docker"])
+        elif isinstance(self.node.os, BSD):
+            raise SkippedException(
```

Comment:
Don't skip in tool level. Raise exception here, and skip in test case level.

Reply:
Any specific reason? Shouldn't a testcase be skipped always if the distro is freebsd?

Reply:
The skip is a test case behavior. For tools, it just describes what happened.